ui/.eslintrc.js

```diff
@@ -24,71 +25,73 @@ module.exports = {
     }
   },
 
-  // These rules were ported over from our common-ui/tslint.json file
+  // Some of these rules were ported over from our common-ui/tslint.json file
   // The rules found here were ported and categorized using the tslint migration roadmap:
   // https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/ROADMAP.md
 
   // The eslint rules can be found here: https://eslint.org/docs/rules/
   rules: {
-    // 'prettier/prettier': 'warn', // Possibly use prettier to handle some formatting
-
-    /* Best Practices */
-    // 'no-multi-spaces': 'warn',
-
-    /* Typescript specific */
-    '@typescript-eslint/explicit-member-accessibility': 'off',
-    // '@typescript-eslint/member-ordering': ['warn', { 'classExpressions': ['method', 'field'] }],
-    '@typescript-eslint/no-empty-interface': 'warn',
-    '@typescript-eslint/no-inferrable-types': ['warn', {ignoreParameters: true}],
-    '@typescript-eslint/no-non-null-assertion': 'warn',
-    // 'prefer-arrow/prefer-arrow-functions': ['warn'], // Lots of 'newable' functions in the code base
-    '@typescript-eslint/type-annotation-spacing': 'warn',
-    '@typescript-eslint/unified-signatures': 'warn',
-
-    /* Style */
-    '@typescript-eslint/prefer-function-type': 'warn',
-    // 'spaced-comment': 'warn',
-    '@typescript-eslint/consistent-type-definitions': ['warn', 'interface'],
-    'no-trailing-spaces': 'warn',
-    'no-undef-init': 'warn',
-    // 'brace-style': ['warn', '1tbs'],
-    // 'simple-import-sort/sort': 'warn',
-    'quotes': ['warn', 'single'],
-    // '@typescript-eslint/semi': 'warn',
-    // 'space-before-function-paren': ['warn', { 'anonymous': 'never', 'named': 'never', 'asyncArrow': 'always' }],
+    /* Code Quality */
```

Comment:
Updated this section after Eng Sync discussion on 5 Jan 2022.

Sorted into Code Quality vs Style sections.  Some of these were judgment calls.  Your opinion may differ.  Fire away!